spanner/api/Spanner.Samples/InsertUsingDmlReturningAsyncPostgre.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright 2023 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START spanner_postgresql_insert_dml_returning]
+
+using Google.Cloud.Spanner.Data;
+using System;
+using System.Threading.Tasks;
+
+public class InsertUsingDmlReturningAsyncPostgreSample
+{
+    public async Task<int> InsertUsingDmlReturningAsyncPostgre(string projectId, string instanceId, string databaseId)
+    {
+        string connectionString = $"Data Source=projects/{projectId}/instances/{instanceId}/databases/{databaseId}";
+
+        using var connection = new SpannerConnection(connectionString);
+        await connection.OpenAsync();
+
+        //Inserts records into Singers table and returns the SingerId ,FirstName and LastName
+        //column of the inserted records using ‘RETURNING SingerId, FirstName, LastName’.
+        //All columns of all the inserted records can be returned using ‘RETURNING *’.
+        using var cmd = connection.CreateDmlCommand("INSERT INTO Singers(SingerId, FirstName, LastName) VALUES(5, 'Bruce', 'Allison') RETURNING SingerId, FirstName, LastName");
```

Comment:
I hope this doesn't conflict with any other tests using Singers table and we should be able to run them in any order. Please confirm.